portality/crosswalks/article_doaj_xml.py

```diff
@@ -215,7 +216,16 @@ def crosswalk_article(self, record, add_journal_info=True):
         if ftel is not None and ftel.text is not None and ftel.text != "":
             ct = ftel.get("format")
             url = ftel.text
-            bibjson.add_url(url, "fulltext", ct)
+
+            try:
+                # url validation
+                result = urlparse(ftel.text)
+                if all([result.scheme, result.netloc, result.path]):
+                    bibjson.add_url(url, "fulltext", ct)
+                else:
+                    raise CrosswalkException(message="Invalid URL: {}".format(ftel.text), inner=e)
+            except Exception as e:
+                raise CrosswalkException(message="Invalid URL: {}".format(ftel.text), inner=e)
```

Comment:
I think we need to revert the changes to this file, so that the crosswalk is not involved in url validation at all

Reply:
done